src/main/resources/com/atlassian/bitbucket/jenkins/internal/trigger/Messages.properties

```diff
@@ -1,4 +1,4 @@
 BitbucketWebhookTrigger.displayname=Bitbucket Server trigger build after push
 BitbucketWebhookTriggerCause.withAuthor=Triggered by Bitbucket webhook due to changes by {0}.
 BitbucketWebhookTriggerCause.withoutAuthor=Triggered by Bitbucket Server webhook.
-BitbucketWebhookMultibranchTrigger.displayname=Bitbucket Server trigger scan after push
\ No newline at end of file
+BitbucketWebhookMultibranchTrigger.displayname=Bitbucket webhook trigger
```

Comment:
@mhenschke-atl are we removing the `Server` part of the name?

Reply:
No word on that yet, but I'll check.